### PR TITLE
check callable

### PR DESCRIPTION
**Pull request by pine3ree**

### CODE REVIEW

File: framework/widgets/ActiveField.php [hunk 1]
```diff
@@ -193,7 +193,7 @@ public function render($content = null)
                 $this->hint(null);
             }
             $content = strtr($this->template, $this->parts);
-        } elseif (!is_string($content)) {
+        } elseif (!is_string($content) && is_callable($content)) {
```
Comment (klimov-paul) [inline framework/widgets/ActiveField.php]: What a sense of extra check?

**Comment by pine3ree** [comment]
@klimov-paul 
Not everything that is not a string a valid parameter for call_user_func. If you use it as a callback you should chek that is callable.

**Comment by klimov-paul** [comment]
So what? At the original code you will get a PHP error, indicating you have passed wrong value. With your solution code will just fail silently. What is the benefit of it?

**Comment by pine3ree** [comment]
Hello @klimov-paul 
i see your point.
In those cases we can either: 1) skip the $content part, 2) return null|empty string 3) throws a customized exception.

I added a custom exception message to help indentify where the wrong input parameter

**Comment by klimov-paul** [comment]
I still don't think an extra check is required here.

**Pull request closed without merging**